todo_app/data/trello_items.py

```diff
@@ -0,0 +1,121 @@
+from flask import session
+import requests, json
+
+# Define a class for Card type 
+class Item:
+        def __init__(self, id, name, statusid, status = 'To Do'):
+            self.id = id
+            self.name = name
+            self.statusid = statusid
+            self.status = status
+
+'''   
+        @classmethod
+        def from_trello_card(cls, id, name, list):
+           return cls( id, name, list )
+'''
+
+
+def get_items(url, trellobid, apikey, apitoken):
+    """
+    Fetches all cards from ToDo list on Trello.
+
+    Returns:
+        list: The list of Cards in All lists on Trello Board.
+    """
+    getcardsurl = url+"1/boards/"+trellobid+"/lists?cards=open&key="+apikey+"&token="+apitoken
+    headers = {
+        'Accept': "application/json"
+    }
+    r = requests.get(url = getcardsurl, headers=headers)
+    carddata = r.json()
+    card_nums = len(carddata)
+    all_cards = []
+    if r.status_code == 200:
+        for num in range(card_nums):
```

Comment:
Rather than looping over the numbers, it can be a bit clearer to just loop over the cards list directly:
```
for card in carddata:
  card_type = card['name']
  ...
```

That said, these aren't actually cards, this is the set of lists unless I'm mistaken!